### ISSUE TITLE
MacOS: Updated High Sierra and now all layers are pink.

### ISSUE THREAD

**Issue by PeacePlays** [mrkite/minutor#157]
Minutor was working prior to Installing Security Update 2019-002 version 10.13.6.

![Screen Shot 2019-03-31 at 9 17 57 AM](https://user-images.githubusercontent.com/49123331/55290381-40466380-5398-11e9-940a-17c5ccd4f1c5.png)

**Comment by EtlamGit** [comment]
Can you take a look into <MacOS User Storage>, Minutor folder?
How many definition files do you find there?

**Comment by PeacePlays** [comment]
After showing package contents and making sure I can see any hidden files and folders I don’t find much of anything. Info.plist, MacOS folder with executable, Pkginfo, Resources folder with  empty.lproj and icon.icns

**Comment by PeacePlays** [comment]
Just to add, I can confirm it works just fine on 10.12.6 (16G1815). Package contents appear identical. But I have not run the specific security update at this time on this machine.

**Comment by EtlamGit** [comment]
My question got miss-formatted by GitHub...
There should be a temporal-app-storage-folder for each app and each user. Not the folder the app is located itself. Updated definition files will be downloaded/stored in this folder.

(And pink Chunks looks like a major problem with the definition files. Pink = unknown)

**Comment by PeacePlays** [comment]
Knowing this, I suspect a permissions error related to the installation of the security update with 10.13. When I deleted the app, I did not delete these resource files. When I get home I will and see if that doesn't fix the issue when I run the app. I suspect it will check for the files and when not there, will add them. Might be that easy.

Found it; It's located in Users: ~Library/Application Support/seancode/Minutor. 

At work on a Mac with 10.12.6 (16G1917) (with the suspected security update installed) and it works fine.

![Screen Shot 2019-04-01 at 8 47 26 AM](https://user-images.githubusercontent.com/49123331/55332377-dfcc2a80-545a-11e9-95a0-daadb424a357.png)
